fission-web-server/README.md

```diff
@@ -41,3 +41,49 @@ $ psql
 ```
 
 Migrations will be performed automatically when running the server
+
+### Using Docker
+
+To mimic the full "fission stack" for local development, you can use the included `docker-compose.yml`. This assumes that you have Docker installed locally:
+
+1. Copy the env.example file to `.env` (these values are used by docker compose). Edit as desired (defaults will work)
+2. Start docker compose: `docker compose up -d`. This will start postgres () and powerdns containers, including a local DNS resolver.
+3. Create the necessary zones (specified in `.env`) e.g.: `docker compose exec dns-auth pdnsutil create-zone fissionapp.test`
+4. Point your local DNS resolver to localhost (on macOS this is under System Preferences > Network > Advanced).
+  
+You can now build / run the haskell server. Make sure `env.yaml` values are updated appropriately. The following env.yaml snippet matches the `env.example` included:
+
+```yaml
```

Comment:
I thought about just updating the env.yaml.example to work locally "out of the box" with this setup. Thoughts?

Reply:
We used to have that! (We also used to have a Docker config, time is a circle 😛). We switched to clear placeholders, because it was confusing for people why some stuff wouldn't actually work (e.g. Heroku endpoint, email, etc)

I'm all for switching it to being "on rails" again, though!

Reply:
switched to Rails 🤔 

(updated the default env.yaml.example)